### ISSUE TITLE
rdesktop should be aware of multiple monitors

### ISSUE THREAD

**Issue by derfian** [rdesktop/rdesktop#205]
rdesktop does not understand multiple monitors, so it sends the combined size of all monitors to the RDP server. We should communicate the monitor layouts both through MCS Connect Initial and through the RDPEDISP display control extension.

**Comment by derfian** [comment]
This is probably a solution to have the login window show up on one screen, reported on issue #87.

**Comment by hean01-cendio** [comment]
It would also be nice if there was an user a option to run rdesktop full screen on one (current) or all monitors to support the use-case:

> As a user I want to run a remote desktop on the left monitor while having the local 
> desktop on the right so that I easily can switch between the remote and local desktop
> while working.

**Comment by uglym8** [comment]
I suggest to consider (at least it's what I'm planning to do) adding a few arguments to "-f" option:

**-f all:multi** - use all available monitors in Multi Monitor Mode

**-f all:span** -  use all available monitors in Span Mode (actually, with some restrictions)
ATM  **"-f"** defaults to this mode. 
Need to check multiple Screens handling though (i.e. if XRandR reports Sreeen 0, Screen 1 etc).

**-f screen_spec:output_spec** e.g. **-f 0:HDMI-2** - use only HDMI-2 output on Screen 0 in fullscreen mode

I could add more args to MultiMonitors Mode (e.g. desktop orientation for each display) but I think
we could discuss this after the above will be implemented.

**Comment by trentasis** [comment]
Multimonitor span is supported in master branch?

**Comment by uglym8** [comment]
Nope, it's still in my TODO list.
I have to find time to finish it.
Sorry for the delay.

**Comment by qoheniac** [comment]
I use a configuration where my left monitor is landscape and my right monitor is portrait, so if I use rdesktop part of the desktop is not displayed.

**Comment by raysgremlin** [comment]
Pi 4 now out with multiply HDMI and still rdesktop not aware of multiple monitors. 
https://www.raspberrypi.org/forums/viewtopic.php?f=36&t=247900
Can we please have this soon ?

**Comment by erichelch** [comment]
Any news about? I'm also interested in a solution - Login-window of the RDP is one half on each monitor....

**Comment by Nindohana** [comment]
Hello ! Any news about this issue ? For me it still open both screen as one wide.
Thanks in advance.

**Comment by dajuly20** [comment]
Yeah try alt + enterAm 17.12.2019 00:55 schrieb Nindohana <notifications@github.com>:Hello ! Any news about this issue ? For me it still open both screen as one wide.
Thanks in advance.

**Comment by qoheniac** [comment]
Maybe you could implement it the same way `xfreerdp` does it with its `/monitors:0,1` and `/multimon` flags?

**Comment by Jack8008** [comment]
Any news on this?

**Comment by raysgremlin** [comment]
I ended up using xfreerdp : Just add your domain,username, password, ip of terminal server, KB is UK keyboard
xfreerdp /d:DOMAIN /u:USERNAME /p:PASSWORD /v:192.19.150.65 /f /kbd:0x00000809 /multimon /clipboard /gdi:hw /sound:latency:400 /printer /cert-ignore

**Comment by qoheniac** [comment]
> Any news on this?

rdesktop isn't maintained anymore. You should use freerdp.
